packages/happy-dom/src/config/HTMLElementConfig.ts

```diff
@@ -128,11 +138,13 @@ export default <
 	},
 	col: {
 		className: 'HTMLTableColElement',
-		contentModel: HTMLElementConfigContentModelEnum.noDescendants
+		contentModel: HTMLElementConfigContentModelEnum.noDescendants,
+		permittedParents: ['colgroup']
 	},
 	colgroup: {
 		className: 'HTMLTableColElement',
```

Comment:
The class name is the same as the previous?

Reply:
This is correct.

If you run the following code in the browser you will see that both are using the `HTMLTableColElement` class:
```javascript
// Outputs "HTMLTableColElement"
console.log(document.createElement('col').constructor.name);

// Outputs "HTMLTableColElement"
console.log(document.createElement('colgroup').constructor.name);
```